meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,121 @@
+# Wayfair plugin: Initial Setup
+
+## Prerequisites
+* [Wayfair API credentials](obtaining_credentials.md).
+
+* Administrative rights on the Plentymarkets system
+
+* [Installation](plugin_installation.md) of the Wayfair plugin - [view release notes](https://github.com/wayfair-contribs/plentymarkets-plugin/releases)
+
+
+## 1. Authorizing the Wayfair Plugin to access Wayfair interfaces
+After the plugin is installed in your Plentymarkets Plugin Set, the plugin must be configured to use the correct credentials when connecting to Wayfair's interfaces.
+
+* **The authorization procedure must be performed for any Plugin Set that contains the Wayfair plugin**.
+* Copying a Plugin Set will copy the authorization information to the new plugin set.
+* An exported or imported Plugin set may include the authorization information.
+
+The authorization steps are as follows:
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+
+2. Locate the Plugin Set that is linked to the client with which Wayfair will be used.
+
+3. Click on the `Edit` button for the desired Plugin set
+
+4. In the Wayfair row of the Plugin set, click on the `Settings` button.
+
+4. In the left-side menu, go to `Configuration` >> `Global Settings`.
+
+5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials
+
+6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md)

```

Comment:
The naming you have have nice labels (i.e. Supplier Settings) vs what I'm seeing (i.e. globalConfigContainerLabel):

![image](https://user-images.githubusercontent.com/22055435/87823293-088b5580-c841-11ea-8ff5-45e35be64699.png)

Is this just an artifact of my system?